tests/src/Kernel/Plugin/Field/FieldFormatter/VideoFormatterTest.php

```diff
@@ -106,4 +110,27 @@ public function testVideoFormatter() {
     $this->assertTrue($children[0] instanceof Video);
   }
 
+  /**
+   * Tests the instant article video formatter when a canonical URL is in play.
+   *
+   * @covers ::viewInstantArticle
+   */
+  function testVideoFormatterCanonicalUrl() {
+    $entity = EntityTest::create([]);
+    // Handy method to populate the field with a real value.
+    // @see FileItem::generateSampleValue()
+    $entity->{$this->fieldName}->generateSampleItems(1);
+
+    // Test with a canonical URL set.
+    $config = $this->config('fb_instant_articles.settings');
+    $config->set('canonical_url_override', 'http://example.com')
```

Comment:
`example.com` is pretty common, and I could see this test passing by mistake. How about making the override `http://example.com/override`?